sentry/src/main/java/io/sentry/NoOpSpan.java

```diff
@@ -0,0 +1,50 @@
+package io.sentry;
+
+import io.sentry.protocol.SentryId;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class NoOpSpan implements ISpan {
```

Comment:
every NoOp object today is a singleton with a `getInstance` method, what's about doing the same here and `NoOpTransaction`?

Reply:
👍